src/i18n/scripts/intl-imports.js

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env node
+const fs = require('fs/promises');
+
+const i18nDir = 'src/i18n';
+const messagesDir = `${i18nDir}/messages`;
+
+async function generateDirectoryIndex(directory) {
+    console.log('generateDirectoryIndex', directory);
```

Comment:
it looks like the logging pattern for the old transifex script was https://github.com/openedx/frontend-platform/blob/9a98efe6ad8e9275078925f85580ef8efc2370c4/src/i18n/scripts/transifex-utils.js#L55

I'm not sure if that's still the preferred way to handle logging

Reply:
I've converted it to that pattern. ESLINT still complain about `console.log` statements`.